### ISSUE TITLE
ImportError: cannot import name 'notebook_content'

### ISSUE THREAD

**Issue by jainrp** [bokeh/datashader#535]
I just installed datashader library and running the examples and got this error for NYC taxi example. Not sure how to fix it... Followed all the instructions for setting up the environments and downloads etc.

```
ImportError                               Traceback (most recent call last)
<ipython-input-12-3d1c4cc2767a> in <module>()
      1 import datashader as ds
----> 2 from datashader.bokeh_ext import InteractiveImage
      3 from functools import partial
      4 from datashader.utils import export_image
      5 from datashader.colors import colormap_select, Greys9, Hot, inferno

~\Anaconda3\envs\ds\lib\site-packages\datashader\bokeh_ext.py in <module>()
     23 if bokeh_version > '0.12.9':
     24     from bokeh.protocol import Protocol
---> 25     from bokeh.embed import notebook_content
     26     from bokeh.embed.notebook import encode_utf8
     27     from bokeh.io.notebook import CommsHandle, get_comms

ImportError: cannot import name 'notebook_content'
```

**Comment by jbednar** [comment]
Looks like bokeh 0.12.11 no longer imports "notebook_content" in "embed/__init__.py".  Not sure why that wasn't making our tests fail, but for now it should work to either downgrade bokeh (``conda install bokeh=0.12.10``) or change that line to ``from bokeh.embed.notebook import notebook_content`` (though in that case there may be other changes needed for bokeh 0.12.11).  We'll need a new datashader release to work with the new Bokeh.

**Comment by jbednar** [comment]
Actually, it turns out that we will also need a new Bokeh release to work with datashader!  There were some issues with the bokeh 0.12.11 release, and we'll soon be releasing datashader 0.6.4 and 
bokeh 0.12.12.  Hopefully will fix it all!

**Comment by jbednar** [comment]
Should be fixed in datashader 0.6.4. But please use bokeh 0.12.10 with it for now, not 0.12.11, or bokeh 0.12.12 when it is released shortly.
